src/protocol/libp2p/kademlia/store.rs

```diff
@@ -1,5 +1,3 @@
-// Copyright 2023 litep2p developers
```

Comment:
nit: Should we do the same for all other files and just keep the copyright without the year?